modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/service.xml

```diff
@@ -35,10 +35,6 @@
 		<finder name="CompanyId" return-type="Collection">
 			<finder-column name="companyId" />
 		</finder>
-
-		<!-- References -->
-
-		<reference entity="User" package-path="com.liferay.portal" />
```

Comment:
@icgsbr why did you remove this? what is the purpose of having this? please make sure nothing will break.

Reply:
The source format was failing with the following message: 

Avoid using reference 'User' for Entity 'DDMContent', use private variables in *ServiceImpl instead: /home/igorcosta/dev/projects/liferay-portal/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/service.xml (SourceCheck:XMLServiceReferenceCheck)

        at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:458)
        at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:291)

Could you help me on make this work? @carolmariaabb

Reply:
This was the only way I managed to make it pass the test and by testing, I saw no errors. Yet, I'm almost sure there's another way of doing it, but idk it.

Reply:
@icgsbr first all of, you need to run `buildService` after removing this (always run `buildSevice` after changing the `service.xml` file). 
It's fine to remove it, but you'll probably need to update the `DDMContentLocalServiceImpl` by adding the private variable as suggested:

```	
@Reference
private UserLocalService _userLocalService;
``` 

I believe this [line](https://github.com/liferay/liferay-portal/blob/918d2a92fea02807d92eec05a2d4ab8903e11aea/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/base/DDMContentLocalServiceBaseImpl.java#L599-L600) will be removed from `DDMContentLocalServiceBaseImpl` after running `buildService`, so this suggested change will fix the compilation error [here](https://github.com/liferay/liferay-portal/blob/918d2a92fea02807d92eec05a2d4ab8903e11aea/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMContentLocalServiceImpl.java#L41).

Reply:
I also managed to see the use of the User on DDMContentLocalServiceImpl and it is defined inside a method, not being a class variable

Reply:
Done